ag/ag.proto

```diff
@@ -203,12 +197,19 @@ message OrgRequest {
     string orgName = 1;
 }
 
-// depending on situation frontend will have all enrolled users or only those that are not members of any group
-// set filterOutGroupMembers go get list of students not belonging to any group for given course
+// Depending on situation front-end will display all the enrolled users or only those that are not members of any group.
+// Set filterOutGroupMembers to get list of students not belonging to any group for given course.
 message EnrollmentRequest {
     uint64 courseID = 1;
     bool filterOutGroupMembers = 2;
-    repeated Enrollment.UserStatus states = 3;
+    repeated Enrollment.UserStatus statuses = 3;
+}
+
+// Used to request courses and enrollments for a given user 
+// where the user has a specific enrollment status.
+message EnrollmentStatusRequest {
```

Comment:
What about:
_EnrollmentStatusRequest is a request for a given user, with a specific enrollment status._

(we should avoid explaining what these message types are used for, at least as the first sentence. Maybe a use case example can be added at the end. Rationale: Some of these types may, in the future, be reused for something more.)

Reply:
Yeah, I see your point. Fixed.

Reply:
I don't see the fix.

Reply:
Just removed the comment. It was supposed to clarify that the same request can be used when fetching courses or enrollments. Otherwise there is no need to explain obvious things. Or maybe the opposite? We could add similar comments for every request type.

Reply:
Well, I don't think we need a comment on everything, at this point at least, but, since we already started adding a comment on this one, I proposed this one as a replacement:
_EnrollmentStatusRequest is a request for a given user, with a specific enrollment status._

(is this not accurate?)

Reply:
Yeah, it is pretty accurate. Added the proposed comment.